### DISCUSSION TITLE
Scratchpads not floating like intended

### DISCUSSION THREAD

**Discussion by Josue-JP** [hyprland-community/pyprland#158] [Q&A]
When I first installed pyprland I had a working scratchpad that I was looking forward to everyday, turns out that the next day I had to wake up with the scratchpads not floating but instead adding another tile on my desktop. I have tried to configure the pyprland.toml but it seems that the scratchpad is not coming from that pyprland.toml, because when I erase everything from the file the scratchpad still loads. please help me. 

pyprland.toml:
[pyprland]
plugins = [
 "expose",
 "fetch_client_menu",
 "lost_windows",
 "magnify",
 "scratchpads",
 "shift_monitors",
 "toggle_special",
 "workspaces_follow_focus",
]

# using variables for demonstration purposes (not needed)
[pyprland.variables]
term_classed = "kitty --class"

[scratchpads.term]
animation = "fromTop"
command = "[term_classed] main-dropterm"
class = "main-dropterm"
size = "50% 50%"
max_size = "1920px 100%"

hyprland.conf: 

###################
###################

source = ~/.config/hypr/hland/keybinds.conf

keybinds.conf:
#pyprland
exec-once = /usr/local/bin/pypr --debug /tmp/pypr.log
bind = ALT, Z, exec, pypr toggle term               # toggles the "term" scratchpad visibility
bind = $mainMod, B, exec, pypr expose                       # exposes every window temporarily or ">
#bind = $mainMod, N, exec, pypr toggle_special minimized     # toggle a window from/to the "minimiz>
#bind = $mainMod SHIFT, N, togglespecialworkspace, minimized   # toggle the "minimized" special wor>
#bind = $mainMod SHIFT, O, exec, pypr shift_monitors +1      # swaps workspaces between monitors
#bind = $mainMod SHIFT, Z, exec, pypr zoom ++0.5             # zooms in the focused workspace
#bind = $mainMod, Z, exec, pypr zoom                         # toggle zooming

**Comment by fdev31** [comment]
Can you check the class of the window ?
